### DISCUSSION TITLE
How do you test a 'Class Under Specification's method, that is being called from the 'Method Under Specification'?

### DISCUSSION THREAD

**Discussion by ghost** [spockframework/spock#1346] [Q&A]
https://stackoverflow.com/questions/67950174/spock-how-to-stub-the-inner-method-call-that-calls-the-same-class

Hey fellows, not sure me mentioning you here (as Spock Contributors) is appropriate - please accepts my apologies if it is not.

Trying to solve the dilemma I have been faced with for the past few days.

Any help is greatly appreciated.
@pniederw @msgilligan @sbglasius

**Comment by kriegaex** [comment]
Thanks for the cross-post. I answered your question on StackOverflow.

**Reply by ghost** [reply]
Thank you, good sir! Sorry did not mean to ping you all. Honestly though this one is not being monitored at all / saw a beta not somewhere.

**Comment by leonard84** [comment]
Welcome to the Spock Discussions.

For future reference, please just post these types of questions only on SO, the same persons that read the Spock discussions are also on SO. As you can see our very active @kriegaex already answered you question over there. We prefer SO, as there is already a large collection of technical Spock Q&A on that site and it makes sense to keep it in one place. This discussion boards is more intended for things that are not issues or questions suited for SO. Unfortunately Github doesn't really allow to display some custom hints to explain this, at the moment.

Btw, just pinging multiple people from the get go is a bit rude, most are busy people that get enough notifications already, and/or haven't been actively involved with Spock for a long time. No need to apologize, just keep it in mind for the future.

**Reply by ghost** [reply]
Thanks for clarifying, guys. 

I will follow the guidelines provided going forward. 

As far as this question - should I delete it from this discussion or should I leave it?
